runtime/syntax/nftables.yaml

```
@@ -1,25 +1,23 @@
 filetype: nftables
 
 detect:
-    filename: "((nftables.conf$)|(nftables.rules.d/)|(nftables.d/)|(nftables.rules$))"
-    header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
+   filename: "(nftables\\.(conf|rules)$|nftables(\\.rules)?\\.d/)"
+   header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
 
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
     - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
     - special: "\\b(elements|hook|policy|priority|type|state)\\b"
-    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th|dport|sport|saddr|daddr)\\b"
+    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th|dport|sport|saddr|daddr|l4proto)\\b"
     - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject|limit|queue)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol.operator: "[<>.&|^!=:;,@]|[!=]|\\b(and|ge|gt|le|lt|or|xor)\\b"
+    - symbol.operator: "[<>.&|^:;,@]|[!=]|\\b(and|ge|gt|le|lt|or|xor)\\b"
```

```suggestion
    - symbol.operator: "[<>.&|^!=:;,@]|\\b(and|ge|gt|le|lt|or|xor)\\b"
```
Isn't it easier to read and shorter?
You can use online regex interpreter like e.g. https://regex101.com/